### PR TITLE
Enhance select test coverage

### LINKED ISSUE #800: Enhance Test Coverage for `@yamada-ui/select`
related: #775

### Description
Enhance test coverage for components with less than 80% coverage to meet or exceed 80%.

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### PR DESCRIPTION
**Pull request by suzukisan22**
Closes #800 

## Description
add test for MultiSelect
coverage raise from 68.08% to 86.38%
mainly, MultiSelect test was insufficient

## Is this a breaking change (Yes/No):

No.

### CODE REVIEW

File: .changeset/flat-badgers-hammer.md
Comment (108yen) [inline .changeset/flat-badgers-hammer.md:1]: Delete thie file.
Changeset is not necessary with test.

File: packages/components/select/tests/select.test.tsx [hunk 1]
```diff
@@ -101,3 +101,79 @@ describe("<Select />", () => {
...
+    const multiSelect = screen.getByTestId("MultiSelect")
+    await user.click(multiSelect)
+    const optionOne = screen.getByTestId("Option-One")
+    await user.click(optionOne)
+
+    const optionTwo = screen.getByTestId("Option-Two")
+    await user.click(optionTwo)
+
+    expect(screen.getByText("One,")).toBeInTheDocument()
+  })
```
Comment (108yen) [inline packages/components/select/tests/select.test.tsx:178]: This test has issued a warning.
Refer to other tests.

**Review by 108yen — CHANGES_REQUESTED** [review]

File: packages/components/select/tests/select.test.tsx (hunk 1 above)
Comment (suzukisan22) [reply packages/components/select/tests/select.test.tsx]: Thank you for your comment!
I fixed this commit. 
https://github.com/yamada-ui/yamada-ui/pull/935/commits/10cac9591567a684c714cc0d16db6344ed1e38f0

File: .changeset/flat-badgers-hammer.md
Comment (suzukisan22) [reply .changeset/flat-badgers-hammer.md:1]: Thank you for your comment!
I fixed in this commit.
https://github.com/yamada-ui/yamada-ui/pull/935/commits/2ab47f2b5480fafc10ca25cd8623e235e5e5c372

**Review by 108yen — APPROVED** [review]
Thank you!

**Pull request merged**